codes/classical/properties/alternant.yml

```diff
@@ -10,8 +10,19 @@ logical: q-ary_digits
 name: 'Alternant code'
 introduced: '\cite{doi:10.1016/S0019-9958(74)80005-7}'
 
-description: 'Stub.'
+description: 'A code with parity check matrix with n columns and t roww of the form
```

Comment:
This is a large code class AFAIK; maybe explain why it's useful to consider this as a class. What are the properties, besides the defn, that unite all the codes in this class? Is there an efficient decoder?